src/aosm/azext_aosm/inputs/helm_chart_input.py

```diff
@@ -136,12 +191,21 @@ def get_defaults(self) -> Dict[str, Any]:
         :raises DefaultValuesNotFoundError: If no default values were found for the Helm chart.
         """
         logger.info("Getting default values for Helm chart input")
+
+        provided_config = None
+
+        if self.default_config_path:
+            if Path(self.default_config_path).exists():
```

Comment:
Using a try/except feels more Pythonic here, but I don't feel strongly if you prefer this style.

Reply:
Although, per comment above about changing `default_config` to `default_config_path`, it might be easier just to move this back to where it was (and then this isn't part of this review :-) )

Reply:
This was how this code was structured originally (I just moved it). But yeah, I will probably move it back